kedro-airflow/tests/test_plugin.py

```diff
@@ -263,6 +268,31 @@ def test_create_airflow_dag_env_parameter_exists(cli_runner, metadata):
         dag_code = [line.strip() for line in f.read().splitlines()]
     assert expected_airflow_dag in dag_code
 
+    _kedro_remove_env(Path.cwd())
+
+
+def test_create_airflow_dag_tags_parameter_exists(cli_runner, metadata):
+    """Test the `tags` parameter"""
+    dag_name = "hello_world"
+    command = ["airflow", "create", "--tags", "tag0", "--env", "remote"]
```

Comment:
Shall we also add at least one test with two tags?

Reply:
Great idea, added!